lib/membrane_rtc_engine/track_synchronizer.ex

```diff
@@ -0,0 +1,90 @@
+defmodule Membrane.RTC.Engine.TrackSynchronizer do
```

Comment:
```suggestion
defmodule Membrane.RTC.Engine.Endpoint.HLS.TrackSynchronizer do
```

Also, add this module in mix.exs -> docs -> group_for_modules -> Endpoints